docs/sql-ref-syntax-aux-show-functions.md

```diff
@@ -60,11 +60,12 @@ SHOW [ function_kind ] FUNCTIONS ( [ LIKE ] function_name | regex_pattern )
   <dd>
     Specifies a regular expression pattern that is used to filter the results of the
     statement.
-    <ul>
-      <li>Only <code>*</code> and <code>|</code> are allowed as wildcard pattern.</li>
-      <li>Excluding <code>*</code> and <code>|</code>, the remaining pattern follows the regular expression semantics.</li>
-      <li>The leading and trailing blanks are trimmed in the input pattern before processing. The pattern match is case-insensitive.</li>
-    </ul>
+     <ul>
+       <li>Except for <code>*</code> and <code>|</code> character, the pattern works like a regular expression.</li>
+       <li><code>*</code> alone matches 0 or more characters and <code>|</code> is used to separate multiple different regular expressions,
+       any of which can match. </li>
+       <li>The leading and trailing blanks are trimmed in the input pattern before processing. The pattern match is case-insensitive.</li>
+     </ul>
```

Comment:
Change to  the same description of ```regex_pattern``` as the one used in SHOW TABLE EXTENDED, SHOW TABLES, SHOW VIEWS.